olp-cpp-sdk-core/include/olp/core/logging/Log.h

```diff
@@ -33,6 +33,21 @@
 #include <sstream>
 #include <string>
 
+/**
+ * @brief Temporary macros to ease EDGE_SDK_ prefix transition review.
```

Comment:
"to ease ... review"?
Remove please, mark them as deprecated.

Reply:
done